manifests/0000_09_cluster-osin-operator_02_crd.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: osins.osin.openshift.io
+  name: authenticationoperatorconfig.authentication.operator.openshift.io
 spec:
-  group: osin.openshift.io
+  group: authentication.operator.openshift.io
   names:
-    kind: Osin
-    listKind: OsinList
-    plural: osins
-    singular: osin
-  scope: Namespaced
+    kind: AuthenticationOperatorConfig
+    listKind: AuthenticationOperatorConfigList
+    plural: authenticationoperatorconfigs
+    singular: authenticationoperatorconfig
+  scope: Cluster
```

Comment:
move this to the top, and in general, make this look exactly like 

```yaml
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  name: kubeapiserveroperatorconfigs.kubeapiserver.operator.openshift.io
spec:
  scope: Cluster
  group: kubeapiserver.operator.openshift.io
  version: v1alpha1
  names:
    kind: KubeAPIServerOperatorConfig
    plural: kubeapiserveroperatorconfigs
    singular: kubeapiserveroperatorconfig
    categories:
    - coreoperators
  subresources:
    status: {}
```

i.e. same categories, need status, etc